docs-src/tutorials/02-usage.md

```diff
@@ -104,7 +104,7 @@ const myTour = new Shepherd.Tour(options);
 
 ##### Tour Options
 
-- `classPrefix`: The prefix to add to all the `shepherd-*` class names.
+- `classPrefix`: The prefix to add to the `shepherd-*` class names assigned to the `document.body`.
```

Comment:
I think it may apply to various class names, not just the body ones. Would have to check which ones it actually applies to.

Reply:
Okay, can you let me know what else is prefixed?

Reply:
It's a lot of things. I would recommend looking for `classPrefix` in the code to see all the places it is used.